scripts/osx-m1-release.sh

```diff
@@ -28,12 +28,12 @@ eval "$(opam env)"
 # Needed so we don't make config w/ sudo
 export HOMEBREW_SYSTEM=1
 
-make setup
-
 # Remove pcre dynamically linked to force MacOS to use static
 # This needs to be done before make setup since it is used there
-ls -l /usr/local/opt/pcre/lib || true
-rm -f /usr/local/opt/pcre/lib/libpcre.1.dylib
+ls -l "$(brew --prefix)"/opt/pcre/lib || true
+rm -f "$(brew --prefix)"/opt/pcre/lib/libpcre.1.dylib
+
+make setup
```

Comment:
I don't understand why doing make setup before or after has any effect on the compilation.
The compilation part is done by make build-core below; the only thing make setup does
is to install opam packages mostly.

Reply:
@lundberg @sww do you know why?

Reply:
Sorry, no ... I'm not familiar with this build script and haven't investigated ... just stumbled on the problem and opened the issue.